CHANGES_NEXT_RELEASE

```diff
@@ -2,6 +2,8 @@
 - Add: subject.condition.notifyOnMetadataChange field to subscriptions, so only-metadata changes doesn't trigger notifications (#3727)
 - Add: new simplifiedNormalized and simplifiedKeyvalues for attrsFormat (#4286)
 - Add: -mqttTimeout (env var ORION_MQTT_TIMEOUT) for MQTT broker connection timeout
+- Add: -disableNgsiv1 (env var ORION_DISABLE_NGSIV1) to disable NGSIv1 API requests
+- Add: -logDeprecate (env var ORION_LOG_DEPRECATE) to log deprecate usages at WARN level
```

Comment:
Should other deprecated stuff be sensible too to this CLI?
https://github.com/telefonicaid/fiware-orion/blob/master/doc/manuals/deprecated.md

Reply:
Yes, the idea is that `-logDeprecated` gets extended with new deprecation usages beyond NGSIv1 requests (but not in this PR :)